lib/util/log.dart

```diff
@@ -105,6 +108,45 @@ class FileLogger {
   static Future<void> clear() async {
     await _logFile.writeAsString('');
   }
+
+  static String _filterLog(String logText) {
+    String filteredLog = logText;
+
+    RegExp combinedRegex = RegExp('("message":".*?")|'
+        '("Authorization: Bearer .*?")|'
+        '("X-Api-Signature: .*?")|'
+        r'(signature: [^,\}]*)|'
+        r'(location: \[.*?,.*?\])|'
+        r'(\\"signature\\":\\".*?\\")|'
+        r'(\\"location\\":\[.*?,.*?\])');
+
+    filteredLog = filteredLog.replaceAllMapped(combinedRegex, (match) {
+      if (match[1] != null) {
+        return '"message":"***"';
+      }
+      if (match[2] != null) {
+        return '"Authorization: Bearer ***"';
+      }
+      if (match[3] != null) {
+        return '"X-Api-Signature: ***"';
+      }
+      if (match[4] != null) {
+        return 'signature: ***';
+      }
+      if (match[5] != null) {
+        return 'location: [***,***]';
+      }
+      if (match[6] != null) {
+        return r'\"signature\":\"***\"';
+      }
+      if (match[7] != null) {
+        return r'\"location\":[***,***]';
```

Comment:
Also, I think maybe we can add another layer to filter all secrets. Since there might be also other places besides these fields.
Eg: 
hex signature: (64 characters hex (32 bytes))
jwt pattern